tools/dm-task-configuration-file-intro.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Task Configuration File of Data Migration
+summary: This document introduces the task configuration file of Data Migration. 
+category: tools
+---
+
+# Task Configuration File of Data Migration
+
+This document introduces the task configuration file of Data Migration --
+[`task.yaml`](https://github.com/pingcap/tidb-tools/blob/docs/docs/dm/zh_CN/configuration/task.yaml), including [Global configuration](#global-configuration) and [Instance configuration](#instance-configuration).
+
+For description of configuration items, see [Task Configuration Argument Description](../tools/dm-task-config-argument-description.md).
+
+## Important concepts
+
+For description of important concepts including `instance-id` and the DM-worker ID, see [Description of important concepts](../tools/dm-configuration-file-overview.md#description-of-important-concepts) 
+
+## Global configuration
+
+### Basic information configuration
+
+```
+name: test                      # The name of the task. Should be globally unique.
+task-mode: all                  # The task mode. Can be set to `full`/`incremental`/`all`.
+is-sharding: true               # Whether it is a sharding task
+meta-schema: "dm_meta"          # The downstream database that stores the `meta` information
+remove-meta: false              # Whether to remove the `meta` information (`checkpoint` and `onlineddl`) before the 
```

Comment:
Whether to remove the `meta` information (`checkpoint` and `onlineddl`) before starting the  synchronization task?